providers/aws/aws_provider.py

```diff
@@ -290,7 +290,11 @@ def generate_regional_clients(service: str, audit_info: AWS_Audit_Info) -> dict:
     # Get json locally
     f = open_file(aws_services_json_file)
     data = parse_json_file(f)
-    json_regions = data["services"][service]["regions"][audit_info.audited_partition]
+    # Check if it is a subservice
+    if service == 'accessanalyzer':
+        json_regions = data["services"]['iam']["regions"][audit_info.audited_partition]
```

Comment:
This is a good point to think because we can't hardcode that there, we need to include that subservices list in the AWS regions file, because we are going to have more subservices to check.